package.json

```diff
@@ -38,6 +42,7 @@
     "gitploy": "^0.1.0",
     "husky": "^0.14.3",
     "lint-staged": "^7.0.0",
+    "marked": "^0.3.17",
```

Comment:
Thanks for adding this dependency version as the warnings were getting annoying. Unfortunately, `marked` is a dependency of a dependency so the true root cause would be resolved there.

Reply:
> Thanks for adding this dependency version as the warnings were getting annoying. Unfortunately, marked is a dependency of a dependency so the true root cause would be resolved there.

Ah, good point. Ignorance on my part, but that makes complete sense.

> One thing I'm curious about, how come package-lock.json wasn't updated?

Hmmm, not sure. I'll fix that.

Reply:
Cool and adding here is the right move because I believe npm's algorithm will ensure the older version won't be installed (unless the dependency of the dependency locked the version)